frontend/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Mapshot",
+  "icons": [
+    {
+      "src": "thumbnail.png",
+      "type": "image/png",
+      "sizes": "144x144"
+    }
+  ],
+  "background_color": "#000000",
+  "display": "standalone",
+  "start_url": "./?",
```

Comment:
I think the `start_url` needs to be removed. Testing it on Android, that creates an app, but it does not work - it just shows "loading", which is likely because it removed all the query parameters from the URL by using the `start_url` instead of the URL of the page (query parameter contain the path of the map to actually load).

Reply:
Ah, I'm running this in static (html output) mode so the viewer was working without a path. My intent here was to strip out the x/y coords, so the map would load in the default position. I can't think of a good way to resolve this, so I'll remove the field :+1:

Reply:
Yeah, that would probably require to generate the manifest dynamically - the server could do it and that would also allow to have a name per map. Whether it is worth it I don't know and in any case, not for now :)